File: Python/UberVan.py
from car import Car

class UberBlack(Car):
        typeCarAccepted = []
        seatsMaterial = []
        
        def __int__(
            self,
            license,
            driver,
            typeCarAccepted,
            seatsMaterial
        ):
            super.__init__(license, driver, typeCarAccepted, seatsMaterial)
            self.typeCarAccepted = typeCarAccepted
            self.seatsMaterial = seatsMaterial
            

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

//Clase hija
class UberVan extends Car {
    Map< String, Map< String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    Integer passenger;

    //Construcctor
    public UberVan(
            //Var padre
            String licence,
            Account driver
            //Var hijo
            //Map< String, Map< String, Integer>> typeCarAccepted,
            //ArrayList<String> seatsMaterial
    ){
        super(licence, driver);
        //this.typeCarAccepted = typeCarAccepted;
        //this.seatsMaterial= seatsMaterial;
    }

    //POLIMOSFISMO
    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 6){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 6 Pasajeros.");
        }
        //super.setPassenger(passenger);
    }

}
